hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandleFactory.java

```diff
@@ -47,6 +50,7 @@ public static <T, I, K, O> HoodieMergeHandle<T, I, K, O> create(
       String fileId,
       TaskContextSupplier taskContextSupplier,
       Option<BaseKeyGenerator> keyGeneratorOpt) {
+    LOG.info("Get updateHandle for fileId " + fileId + " and partitionPath " + partitionPath + " at commit " + instantTime);
```

Comment:
Are these logs really necessary? If so, please consider logging in debug mode. Same for all logs.